tests/data/cc/dataset/toy_dataset.json

```diff
@@ -226,7 +222,9 @@
           "indirectly_referenced_by": null,
           "indirectly_referencing": null
         },
-        "scheme_data": null
+        "scheme_data": null,
+        "protection_profiles": null,
+        "eal": null
```

Comment:
Why are these entries `null`, while the `"protection_profile_links"` entry has a single PP element? Is it that the toy dataset was not processed fully?

Reply:
Yes. Is in a state after `get_certs_from_web()` IMO. These are heuristics attributes filled-in only in the `analyze_dset()` step.